README.md

```diff
@@ -4,18 +4,77 @@ A collection of helpers for building a rich text editor (WYSIWYG) with [ProseMir
 
-There is a quirk with the PR that is automatically generated at the moment where it cannot trigger the CI tasks to run. In order to run these, push an empty commit to the PR:
+### Triggering a new release
 
+This repo uses [changesets](https://github.com/changesets/changesets) for versioning. When you want to trigger a new release:
+- run `yarn changeset add` from your local terminal to generate a changeset.
+    - Select the [semver change type](https://semver.org/) (ie: `major`, `minor`, `patch`)
+    - Write a concise summary of the change -  this is used in the changelog
+- Commit the generated file and merge the PR
+    - This will automatically trigger a release PR
+- Checkout to the release PR (`changeset-release/main`) and push up an empty commit
 ```bash
 git checkout changeset-release/main
 git commit -m "Empty commit to trigger CI tasks" --allow-empty
 git push origin
 ```
+- Approve the PR and merge into `main`
+
+### Why the empty commit?
+
+This is due to a quirk with the PR that is automatically generated. At the moment it cannot trigger the CI tasks to run. In order to run these, push an empty commit to the PR before merging.
+
+
+## Local development
+
+As there is currently no storybook attached to the Rich Text Toolkit, for ease of local development we would recommend either of the following: 
+- run `yarn build` and copy your `/dist` folder into your local `@kaizen/rich-text-editor` package - this is only really helpful to spot check your current change
+- Use [yarn link](https://classic.yarnpkg.com/en/docs/cli/link) or [yacl](https://github.com/wclr/yalc) to create a dependency on your local development environment with the `@kaizen/rich-text-editor`
+  - **Caveat:** there has been some issue getting `yarn link` to work from machine to machine so we recommend using `yacl`
+
+## Using the rich-text-toolkit
+
+The Toolkit was built out with the intention to be used in `@kaizen/rich-text-editor`, which handles the UI and logic around interactive states. While it is recommended that you leverage the Kaizen component, you can utilize the Toolkit to build out a custom solution if modelled similarly.
+
+If you have spoken to the Design System Team and need a bespoke solution, we advise contributing to the Toolkit to build the core functionality and importing that into your project.
+
+
+## Architecture
+
+### Diagram
+
+![RTE Diagram](/docs/assets/rich-text-toolkit-at-a-glance.png)
+*Note: this diagram was created in the planning stage and while it mostly represent the overall structure, there may be some inconsistencies as development continues.*
+
+### Core
+
+Contains the essential functionality to build out the RTE, Including:
+- Initializing the editor view (`/create.ts`)
+    - `createRichTextEditor` creates a new instance of the RTE that can dispatch commands and transactions.
```

Comment:
I think this is giving the reader the impression that this is what they should use to create their RTE, but in practice everyone is using the `useRichTextEditor` hook, and this is just acting an internal function for that hook.